### ISSUE TITLE
env var to create admin user seems broken in V5

### ISSUE THREAD

**Issue by AmsellemJoseph** [mautic/docker-mautic#261]
Hey, I have an other question, 

Is there a way like for the v4 to pass some env var like
MAUTIC_ADMIN_EMAIL
MAUTIC_ADMIN_USERNAME
MAUTIC_ADMIN_PASSWORD

so when we deploying a new instance, the software will be ready to use with the right credentials.
I used to use it in the v4 and it was working fine.

Thx a lot

**Comment by mollux** [comment]
@AmsellemJoseph yes, see https://github.com/mautic/docker-mautic?tab=readme-ov-file#configuration-and-customizing

**Comment by mollux** [comment]
@AmsellemJoseph were you able to proceed setting those vars based on the documentation?

**Comment by AmsellemJoseph** [comment]
Hey @mollux , no, but I found a way, i'm harcoding the values in the local.php, and after i'm using the cli with mautic:install

**Comment by lazyest** [comment]
still nothing changed as I see

**Comment by IonutOjicaDE** [comment]
Hey there! I'm helping the new maintainers of this repository. It was neglected for some time but that changes now! Could you please re-test if you are still getting this issue so we could focus our limited time on the issues that are still relevant? If there will be no response we'll close this issue in 1 week. It can always be re-opened later on.

**Comment by cibero42** [comment]
Hello, we'll have a look on it after dealing with some priorities!

**Comment by cibero42** [comment]
Hello,

This issue refers to v4, which isn't supported anymore. I'm closing this issue, but you can reopen it if you chack that the same happens on v5!
